lib/std/net/test.zig

```diff
@@ -4,6 +4,27 @@ const net = std.net;
 const mem = std.mem;
 const testing = std.testing;
 
+test "parse and render IP addresses at comptime" {
```

Comment:
hm, I wonder if this test should be skipped in WASI? i.e. like in the `parse and render IPv4/6 addresses` tests below?
```zig
if (builtin.os.tag == .wasi) return error.SkipZigTest;
```
Not sure how to test it in WASI. Let me know if it's better to skip for now.

Reply:
Ended up adding wasi skip to be in line with other similar tests below.